emojirades/commands/gamestate_commands/inferred_correct_guess_command.py

```diff
@@ -71,7 +82,7 @@ def execute(self):
         else:
             emoji = random.choice(self.other_emojis)
 
-        emoji = f" {emoji}"
+        emoji = f" :{emoji}:"
```

Comment:
so erm.. should these be reflected in the other class `correct_guess_command.py` as well?

Reply:
That's a dam good question

Reply:
ok pushed!